cachi2/core/models/sbom.py

```diff
@@ -93,3 +96,173 @@ class Sbom(pydantic.BaseModel):
     def _unique_components(cls, components: list[Component]) -> list[Component]:
         """Sort and de-duplicate components."""
         return unique_sorted(components, by=lambda component: component.key())
+
+    def to_spdx(self) -> "SPDXSbom":
+        """Convert a CycloneDX SBOM to an SPDX SBOM."""
+        return SPDXSbom(
+            packages=[
```

Comment:
A general rule of thumb is this: if your list comprehension spans more than one line you probably need a good old `for` loop. Not a blocker, but would help here.